Library/Formula/libodfgen.rb

```diff
@@ -1,5 +1,5 @@
 class Libodfgen < Formula
-  homepage "http://sourceforge.net/p/libwpd/wiki/libodfgen/"
+  homepage "https://sourceforge.net/p/libwpd/wiki/libodfgen/"
```

Comment:
I can't reproduce here. Three different browsers, two different devices. All going direct to plaintext for me. Will leave this open for the other maintainers to check.